src/main/java/ca/ciccc/BankAccount.java

```diff
@@ -123,7 +135,24 @@ public boolean transactionFee(double fee) {
     public boolean transfer(double amount, BankAccount other) {
         // TODO 3: Your code goes here.
 
-        return false;
+        if (balance <= 5){
+            return false;
+        }
```

Comment:
[FYI] The condition in the next if statement includes this condition. This if statement is unnecessary.